### DISCUSSION TITLE
Does Ergo have a plan to support OTP25/OTP26?

### DISCUSSION THREAD

**Discussion by leonlee2013** [ergo-services/ergo#123] [Q&A] [answered]
[OTP-26 is scheduled to be released on May 1, 2023](https://github.com/erlang/otp/milestone/162)
If use Ergo to communicate with Erlang, Erlang cannot be upgraded to OTP25 or above, which is a limitation to Ergo's development. 
Is there any plan for Ergo to support OTP25/26?

**Comment by halturin** [comment]
Thanks for the question. Sure, we will support them. The support of OTP/25 will be added in the 2.2.2 release (~Feb/Mar 2023).

**Reply by leonlee2013** [reply]
Great, I'm looking forward to the release of 2.2.2！！！

**Comment by halturin** [comment]
created issue #124

**Comment by halturin** [comment, marked as answer]
Just turned out that support of OTP 25 is almost done, so we will try to include it in the coming release 2.2.1 (by the end of this year)

**Reply by leonlee2013** [reply]
It's so cool.
